packages/components/psammead-social-embed/src/utilities/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Returns a string whose word or words have been transformed to Title Case.
+ * @param {String} text A word or words.
+ */
+const toTitleCase = text =>
```

Comment:
As discussed during the swarm review we think this function and the function on line 20 could potentially be replaced with a simple mapping in the top level component to reduce the amount of JS needed.